clients/drcachesim/tests/record_filter_as_traced.templatex

```diff
@@ -0,0 +1,6 @@
+Output .* entries from .* entries.
+Schedule stats tool results:
+.*
+Core #0 schedule: .*
+Core #1 schedule: .*
+Core #2 schedule: .*
```

Comment:
There should be 7 cores right? Maybe check for all?

Reply:
Done.